src/indexer/main.cpp

```diff
@@ -87,6 +87,11 @@ int main(int argc, char *argv[])
 	ApplicationSettings* appSettings = ApplicationSettings::getInstance().get();
 	appSettings->load(FilePath(UserPaths::getAppSettingsPath()));
 	LogManager::getInstance()->setLoggingEnabled(appSettings->getLoggingEnabled());
+    Logger* logger = LogManager::getInstance()->getLoggerByType("FileLogger");
```

Comment:
Please make sure to adjust the indentation to tabs instead of spaces.

Also you could move the initialization of `logger` into the if like:
```
if (Logger* logger = LogManager::getInstance()->getLoggerByType("FileLogger"))
{}
```
or even
```
if (FileLogger* logger = dynamic_cast<FileLogger*>(LogManager::getInstance()->getLoggerByType("FileLogger")))
{}
```
, so that the variable is only visible in the `if` scope.

Reply:
Oh, wait a second. This is not even the correct path to put this. The `src/indexer/main.cpp` file is the entry point of the indexer executable, that gets started (multiple times) when Sourcetrail indexes a project. Please move these changes to `setupLogging()` in `src/app/main.cpp`.

Reply:
Or even better: move it to `Application::loadSettings()`, because in `setupLogging`, the application setting have not even been loaded.

Reply:
Alright will do!